content/blog/2018-03-timers.md

```diff
@@ -117,9 +117,9 @@ And with that, have a great weekend!
 [new release]: https://crates.io/crates/tokio/0.1.5
 [wheel]: http://www.cs.columbia.edu/~nahum/w6998/papers/sosp87-timing-wheels.pdf
 [2]: https://crates.io/crates/tokio-timer/0.2.0
-[delay]: https://docs.rs/tokio/0.1/tokio/timer/struct.Delay.html
-[timeout]: https://docs.rs/tokio/0.1/tokio/timer/struct.Timeout.html
-[interval]: https://docs.rs/tokio/0.1/tokio/timer/struct.Interval.html
+[delay]: https://docs.rs/tokio/latest/tokio/timer/struct.Delay.html
+[timeout]: https://docs.rs/tokio/latest/tokio/timer/struct.Timeout.html
+[interval]: https://docs.rs/tokio/latest/tokio/timer/struct.Interval.html
```

Comment:
This breaks the links. There's no timer module in the latest Tokio release. If possible, I would prefer to add `rel="nofollow"` to the links instead. You might need to use html tags in the markdown to do that.